cmd/antrea-agent/options.go

```diff
@@ -423,6 +427,27 @@ func (o *Options) setK8sNodeDefaultOptions() {
 	}
 }
 
+func (o *Options) validateEgressConfig(encapMode config.TrafficEncapModeType) error {
+	if !features.DefaultFeatureGate.Enabled(features.Egress) {
+		return nil
+	}
+	if encapMode != config.TrafficEncapModeEncap {
+		klog.InfoS("The Egress feature gate is enabled, but it won't work because it is only applicable to the %s mode", config.TrafficEncapModeEncap)
```

Comment:
I am not sure this works. `klog.InfoS` expects key-value pairs, I don't think it supports string formatters?

See https://go.dev/play/p/GODntNgchuC

Reply:
+1

Reply:
thanks, fixed